src/Uno.UI/UI/Xaml/Controls/TimePicker/TimePickerFlyout.iOS.cs

```diff
@@ -1,58 +1,79 @@
 ﻿#if XAMARIN_IOS
+
 using System;
 using System.Collections.Generic;
 using System.Text;
+using Uno.UI;
 using Uno.UI.Common;
+using Uno.UI.Helpers.WinUI;
+using Windows.UI.Xaml.Automation;
 using Windows.UI.Xaml.Controls.Primitives;
 using Windows.UI.Xaml.Data;
 
 namespace Windows.UI.Xaml.Controls
 {
-	//TODO: should inherit from PickerFlyoutBase
 	public partial class TimePickerFlyout : Flyout
 	{
-		private bool _hasAcceptButton = false;
+        private bool AlreadyCancelledOrSaved { get; set; }
 
 		public TimePickerFlyout()
 		{
 			InitializeContent();
 		}
 
-		private void InitializeContent()
+		private void onTap(object sender, Input.PointerRoutedEventArgs e) => e.Handled = true;
+
+		protected internal override void Close()
 		{
-			Content = new TimePickerSelector();
-			BindToContent(nameof(Time));
-			BindToContent(nameof(ClockIdentifier));
+			var timePicker = _presenter as TimePickerFlyoutPresenter;
+			var headerTapZone = timePicker.FindName("HeaderTapZone") as Border;
+			headerTapZone.PointerPressed -= onTap;
 
-			AttachAcceptCommand((TimePickerSelector)Content);
+            if (AlreadyCancelledOrSaved == false)
+            {
+                var timeSelector = Content as TimePickerSelector;
 
-			this.Closed += (_, __) =>
-			{
-				// If no accept button is explicitly part of the template, then update the set Time on light dismiss
-				if (!_hasAcceptButton)
-				{
-					(Content as TimePickerSelector)?.UpdateTime();
-				}
-			};
+                if (timeSelector != null)
+                    timeSelector.Cancel();
```

Comment:
Need brackets here + need tabs, not spaces.

Reply:
Find this convention overkill. Mostly half the code blocks are one liners, more so in well refactored code 

Might make it easier for code reviews/compare and merge but it simply add to much useless lines of code

i'll follow the convention but ..